erpnext/manufacturing/doctype/bom/bom.js

```diff
@@ -331,7 +331,7 @@ frappe.ui.form.on("BOM", {
 			});
 		});
 
-		if (has_template_rm) {
+		if (has_template_rm && has_template_rm.length) {
```

Comment:
PS: this was an issue in framework; still makes sense to fix it here also: https://github.com/frappe/frappe/pull/15734